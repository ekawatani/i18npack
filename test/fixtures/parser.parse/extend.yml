test: !extend
  _base: test/fixtures/parser.parse/basic.yml
  foo: override_foo_text
  one: text1
  two: text2