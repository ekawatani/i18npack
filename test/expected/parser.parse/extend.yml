test:
  foo: override_foo_text
  bar: bar_text
  baz:
    - arrayVal1
    - arrayVal2
    - arrayVal3
  one: text1
  two: text2